examples/plotting/ImageAnimatorWCS_Example.py

```diff
@@ -0,0 +1,40 @@
+"""
+================
+ImageAnimatorWCS
+================
+
+This example shows a simple visualization for an ImageAnimatorWCS object
+"""
+import matplotlib.pyplot as plt
+from itertools import product
+import numpy as np
+
+import astropy
+import astropy.units as u
+
+import sunpy
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE, AIA_193_IMAGE
+from sunpy.visualization.animator import ImageAnimatorWCS
```

Comment:
Do we need all of these imports? Are they all used?

Reply:
yes, as I can see

Reply:
I can't see where `numpy`, `astropy` or `sunpy` used?